src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -170,4 +171,26 @@ private function createNode(\DOMDocument $dom, string $nodeName): \DOMElement
 
         return $node;
     }
+
+    private function addRandomTestsOrderAttributes(string $version, \DOMXPath $xPath): void
+    {
+        if (!version_compare($version, '7.2', '>=')) {
+            return;
+        }
+
+        $this->updateOrAddAttribute('executionOrder', 'random', $xPath);
```

Comment:
I feel like this may be suboptiomal for some projects because people can get used to hidden dependencies (e.g. tests executed in order of definition, communicating through instance's variable; heck, even PHPUnit has such nasty example [right in the manual](https://phpunit.readthedocs.io/en/7.4/fixtures.html)!). Therefore I feel like it will be more appropriate to check if there's already an `executionOrder` attribute with any value (it could be `default` or `reverse`), not overwriting it if it is set.

Reply:
done